Assets/Scripts/MusicController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MusicController : MonoBehaviour
+{
+
+    public enum MusicType {
+        AMBIENT,
+        FIGHT
+    };
+
+    private AudioSource audioSource;
+    public AudioClip fightTheme; 
```

Comment:
public fields are PascalCase :triggered:

Reply:
will fix, did not know about this convention :)